django_opensearch_dsl/documents.py

```diff
@@ -115,14 +105,41 @@ def get_indexing_queryset(
             if verbose:
                 stdout.write(f"{action} {model}: {round(i / count * 100)}% ({self._eta(start, done, count)})\r")
 
-            for obj in qs[i : i + chunk_size]:
+            for obj in qs.iterator(chunk_size=chunk_size):
                 done += 1
+                if done % chunk_size == 0:
+                    stdout.write(f"{action} {model}: {round(done / count * 100)}% ({self._eta(start, done, count)})\r")
                 yield obj
```

Comment:
The outer `while` is useless here since the `for` will already loop over everything.